### DISCUSSION TITLE
How to make the background of navview transparent?

### DISCUSSION THREAD

**Discussion by luluxiaoyu** [Kinnara/ModernWpf#452] [General]
How to make the background of navview transparent?  
I added a background to the form, but the background of the navview navigation bar menu cannot be transparent, which is not beautiful.

**Comment by AntiHero7** [comment]
There are some brushes that affects the background. You can 'overload' them in your xaml by putting new brushes as resources with the same key as the brushes in this library and setting their color to transparent. Here is example in MainWindow.xaml considering that you have Grid and the NavigationView inside that grid.:
```xaml
<Grid>
  <Grid.Resources>
      <SolidColorBrush x:Key="NavigationViewTopPaneBackground" Color="Transparent" />
      <SolidColorBrush x:Key="NavigationViewDefaultPaneBackground" Color="Transparent" />
      <SolidColorBrush x:Key="NavigationViewExpandedPaneBackground" Color="Transparent" />
  </Grid.Resources>
  <ui:NavigationView Name="NavView">
      ...
  </ui:NavigationView>
</Grid>
```
Maybe you can add those brushes as resources to the navigation view itself, that would probably work too. I just remembered that I did it that way some time ago before moving them to separate resource dictionary. Anyway, I hope this will help you :)

**Comment by luluxiaoyu** [comment]
> 

thank you very much! this is useful!
